app/src/main/AndroidManifest.xml

```diff
@@ -23,7 +23,8 @@
       android:label="@string/app_name"
       android:supportsRtl="true"
       android:theme="@style/AppTheme"
-      android:networkSecurityConfig="@xml/network_security_config">
```

Comment:
Why have you removed this line?

Reply:
@mhutti1 It was due to some merge issue. Anyway, I have resolved them now.